internal/types/types.go

```diff
@@ -0,0 +1,46 @@
+package types
+
+import (
+	"errors"
+	"time"
+)
+
+var (
+	// Placeholder until check for valid twitter user function is implemented
+	// TODO: implement twitter username check (api? scraping?)
+	INVALID_TWITTER_USER = "dne"
```

Comment:
We don't need to worry about this functionality for now as I think you're right, it's much harder than it should be and probably will incur  a cost.